opennms-model/src/main/java/org/opennms/netmgt/model/OnmsMonitoredServiceDetail.java

```diff
@@ -73,6 +75,7 @@ public OnmsMonitoredServiceDetail(OnmsMonitoredService service) {
         m_statusCode = service.getStatus();
         m_status = service.getStatusLong();
         m_id = service.getXmlId();
+        m_ipInterfaceId = service.getIpInterface().getId();
```

Comment:
Do we need to store this as a field, can't we just return that in the `getIpinterfaceId()` method?

Reply:
done